src/main/java/racingcar/CarName.java

```diff
@@ -0,0 +1,41 @@
+package racingcar;
+
+import java.util.Objects;
+
+public class CarName {
+    private static final String NUMBER_RANGE_ERROR_MESSAGE = "1~5 사이의 값을 입력해주세요.";
+
+    private String carName;
+
+    public CarName(String carName) {
+        validateCarName(carName);
```

Comment:
```suggestion
        checkCarNameLength(carName);
```

로 변경하는것과 무슨 차이가 있을까요? 🤔

Reply:
원래는 자동차 이름의 길이 체크뿐만 아니라 중복 체크까지validateCarName 안에서 수행하려고 했습니다.

그런데 중복 체크를 CarNameRepository 클래스에서 하게 되었고,

이에 따라 수정했어야 했는데 굳이 함수 하나로 할 수 있는 일을 두 개로 해버렸군요..!